R/hiper_glm.R

```diff
@@ -1,17 +1,47 @@
 #' @export
-hiper_glm <- function(design, outcome, model="linear") {
+hiper_glm <- function(design, outcome, model="linear", option = list()) {
 
   supported_model <- c("linear", "logit")
 
   if (!(model %in% supported_model)) {
     stop(sprintf("The model %s is not supported.", model))
   }
 
-  warning("`hiper_glm` is yet to be implemented.")
+  coef_estimate <- NULL
+
+  if (!is.null(option[["mle_solver"]])) {
+
+    if (option[["mle_solver"]] == "BFGS") {
+      # message("Using BFGS\n")
+
+      op <- stats::optim(par = rnorm(dim(design)[2]),
```

Comment:
-1

Don't use unseeded random inits, otherwise the behavior of the function can change each time you run it. A constant vector will suffice.